### PR TITLE
Chinese blog of comparing MP and Spring

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Fixes: #2638

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
I tried to add a few images as well. However, the images was not displayed in the blog. Please take a look to see whether I have done wrong.

File: posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc
Comment (dmuelle) [inline posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc]:
```suggestion
image::/img/blog/spring_starter.png[SpringBoot starter, width=70%,align="center"]
```

File: posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc
Comment (dmuelle) [inline posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc]:
```suggestion
image::/img/blog/MP_starter.png[MicroProfile Starter, width=70%,align="center"]
```

**Comment by dmuelle** [comment]
Hi Emily- I found a couple typos in the image URLs. I fixed them and the images are showing in the PR preview now.

**Comment by Emily-Jiang** [comment]
Awessome, thanks @dmuelle !

**Comment by kinueng** [comment]
There are a few things we need to do on the `openliberty.io` website code to support this blog post. I will push this through to production when `openliberty.io` is ready for this PR.

**Review by kinueng — CHANGES_REQUESTED** [review]
@kinueng and team need to deliver code before this PR can be merged.

This PR is blocked by https://github.com/OpenLiberty/openliberty.io/issues/2931

**Pull request merged**
